src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.swift

```diff
@@ -48,27 +46,126 @@ class CheckinQRCodeScannerViewController: UIViewController {
 
 	// MARK: - Private
 
+	private var focusView = QRScannerFocusView()
```

Comment:
why still no let ?

Reply:
Forgot to commit it sorry, just pushed again